scripts/appleseedMaya/renderGlobals.py

```diff
@@ -230,13 +230,27 @@ def addRenderGlobalsScriptJobs():
     g_nodeRemovedCallbackID = om.MDGMessage.addNodeRemovedCallback(
         __nodeRemoved)
 
+    # This is evalDeferred so it doesn't get
+    # called before createMayaSoftwareCommonGlobalsTab
+    python_script = "import appleseedMaya.renderGlobals; appleseedMaya.renderGlobals.currentRendererChanged()"
     mc.scriptJob(
         attributeChange=[
             "defaultRenderGlobals.currentRenderer",
-            "import appleseedMaya.renderGlobals; appleseedMaya.renderGlobals.currentRendererChanged()"
+             lambda: mc.evalDeferred(python_script, lowestPriority=True),
         ]
     )
 
+    # For opening new scene with default renderer appleseed
+    mc.scriptJob(
+        event=[
+            'NewSceneOpened',
+            lambda: mc.evalDeferred(python_script, lowestPriority=True),
+        ]
+    )
+
+    # For maya startup empty scene default renderer appleseed
```

Comment:
*Maya*

Missing "with" maybe? Not sure how to parse this sentence otherwise :)